test/gen-testfiles

```diff
@@ -18,11 +23,17 @@ mockery -name Interface -print \
     |sed 's/type Interface/type Ringpop/g' \
     > "$mock_directory"/ringpop.go
 
-echo
-
 mockery -name NodeInterface -recursive -print \
     |sed 's/_m \*NodeInterface/_m \*SwimNode/g' \
     |sed 's/type NodeInterface/type SwimNode/g' \
     > "$mock_directory"/swim_node.go
 
 mockery -case=underscore -dir router -recursive -output "$mock_directory" -name ClientFactory
+
+# make a EventListenerMock for forward (can't use on from $mock_directory because of circular dependency)
+mockery -name EventListener -dir events -recursive -print \
+    |sed 's/package mocks/package forward/g' \
+    > "${0%/*}/../forward/mock_event_listener_test.go"
```

Comment:
it is to work around strange mockery behavior and cyclic recursion.

Normally mockery makes the mocks in the `mocks` package. But the interface for Ringpop imports the `forward` package, preventing the `forward` package its self to import our shared mocks package.

Therefor I instruct mockery to print the generated code to `stdout` and replace the package name in the generated code with the package name I am going to write to file to.

Because the file will be written to a `_test.go` file it is only available to the `forward` package, and only when run with `go test`. During `go build` this generated file will be ignored.